comfy/model_management.py

```diff
@@ -81,6 +81,10 @@ class CPUState(Enum):
 if args.cpu:
     cpu_state = CPUState.CPU
 
+def is_nvidia():
+    global cpu_state
```

Comment:
(nit) you do not need to declare a `global` to read a variable, only to _write_ it, which is not happening here. (This mistake is all over the existing code here tbf)